config.m4

```diff
@@ -365,11 +365,11 @@ PHP_ARG_WITH(libmongoc, whether to use system libmongoc,
   if test "$PHP_LIBMONGOC" != "no"; then
     AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
     AC_MSG_CHECKING(for libmongoc)
-    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongoc-1.0 && $PKG_CONFIG --exists libmongoc-priv; then
+    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongoc-1.0; then
       if $PKG_CONFIG libmongoc-1.0 --atleast-version 1.3.3; then
-        LIBMONGOC_INC=`$PKG_CONFIG libmongoc-priv --cflags`
```

Comment:
libbson never had `-priv`, so no, thats not needed.
Good spot on `--atleast-version 1.3.3` should be bumped to `1.4.0` if you decide on making that the minimum version.

We did not try to compile current php master against `1.3.3` as it currently declares as its minimum. I guess that is not correct today before or after this patch.
Before this patch, you cannot build against system mongoc 1.4, even though you do seem to require it.